res/css/views/dialogs/_DevtoolsDialog.scss

```diff
@@ -28,7 +28,7 @@ limitations under the License.
 .mx_DevTools_content {
     margin: 10px 0;
     overflow-y: auto;
-    height: calc(100% - 124px); // 58px for buttons + 50px for header + 8px margin around
+    height: calc(100% - 130px); // 58px for buttons + 50px for header + 11px margin around
```

Comment:
hmm, where is that 11px margin set? That's a really strange margin size

Reply:
> hmm, where is that 11px margin set? That's a really strange margin size

I knew this was going to be asked😆. I seriously couldn't figure out what the previous comment actually was insinuating as well

Reply:
Nonetheless, i realized if you summed all the values it amounts to 124px for the previous comment, so i decided to do same as well to mine to make it 130px

Reply:
No point having a comment if it is wrong/doesn't make sense.
Equally I can't reproduce the issue here

<img width="778" alt="image" src="https://user-images.githubusercontent.com/2403652/164459382-54717b19-5703-492e-98b2-781c6b5a987e.png">

Reply:
I can't reproduce this in Firefox on Linux.

Reply:
I guess i should close the issue then. It was the same across all my browsers on windows.

Reply:
my screenshot is from FF on MacOS, I can't reproduce in Element Desktop on MacOS either EXCEPT when the scrollbar kicks in but then the misalignment is in the opposing direction

<img width="785" alt="image" src="https://user-images.githubusercontent.com/2403652/164464129-95b311b6-d365-4b15-a637-40e8a28c5fd8.png">

Reply:
I only tried it with element-web  and i am using windows. Let me close my PR and issue. Thanks